depends/hosts/mingw32.mk

```diff
@@ -1,3 +1,7 @@
+ifneq ($(shell which $(host)-g++-posix),)
+mingw32_CXX := $(host)-g++-posix
```

Comment:
Do we need `:=` here? Can we just do `=`?

Reply:
> Do we need `:=` here?

No, we don't.

> Can we just do `=`?

Yes, we can.

But I do prefer simply expanded `make` variables to recursively expanded ones. They make debugging and reasoning about `Makefile`s much easier. Actually, I think the choice of a simply expanded variable is the default, while the usage of a recursively expanded variable requires justification.

In this particular case the `host` variable is simply expanded, and all code is placed in an included file. Honestly, I cannot see advantages for `mingw32_CXX` being a recursively expanded variable here.

Or did I miss something?